gbpservice/nfp/config_orchestrator/agent/loadbalancerv2.py

```diff
@@ -0,0 +1,457 @@
+        if name.lower() == 'loadbalancer':
+            lb_id = kwargs['loadbalancer']['id']
+            kwargs['loadbalancer'].update({'description': str(description)})
+            nfp_context = {'network_function_id': nf['id'],
+                           'loadbalancer_id': kwargs['loadbalancer']['id']}
+        elif name.lower() == 'listener':
+            lb_id = kwargs['listener'].get('loadbalancer_id')
+            kwargs['listener']['description'] = str(description)
+        elif name.lower() == 'pool':
+            lb_id = kwargs['pool'].get('loadbalancer_id')
+            kwargs['pool']['description'] = str(description)
+        elif name.lower() == 'member':
+            pool = kwargs['member'].get('pool')
+            if pool:
+                lb_id = pool.get('loadbalancer_id')
+            kwargs['member']['description'] = str(description)
+        elif name.lower() == 'healthmonitor':
+            pool = kwargs['healthmonitor'].get('pool')
+            if pool:
+                lb_id = pool.get('loadbalancer_id')
+            kwargs['healthmonitor']['description'] = str(description)
+        else:
+            kwargs[name.lower()].update({'description': str(description)})
+            lb_id = kwargs[name.lower()].get('loadbalancer_id')
+
+        args = {'tenant_id': tenant_id,
+                'lb_id': lb_id,
+                'context': context,
+                'description': str(description)}
+
+        ctx_dict, rsrc_ctx_dict = self.\
+            _prepare_resource_context_dicts(**args)
+
+        nfp_context.update({'neutron_context': ctx_dict,
+                            'requester': 'nas_service'})
+        resource_type = 'loadbalancerv2'
+        resource = name
+        resource_data = {'neutron_context': rsrc_ctx_dict}
+        resource_data.update(**kwargs)
+        body = common.prepare_request_data(nfp_context, resource,
+                                           resource_type, resource_data,
+                                           description['service_vendor'])
+        return body
+
+    def _post(self, context, tenant_id, name, nf, **kwargs):
+        body = self._data_wrapper(context, tenant_id, name,
+                                  'CREATE', nf, **kwargs)
+        transport.send_request_to_configurator(self._conf,
+                                               context, body, "CREATE")
+
+    def _delete(self, context, tenant_id, name, nf, **kwargs):
+        body = self._data_wrapper(context, tenant_id, name,
+                                  'DELETE', nf, **kwargs)
+        transport.send_request_to_configurator(self._conf,
+                                               context, body, "DELETE")
+
+    def _fetch_nf_from_resource_desc(self, desc):
+        desc_dict = ast.literal_eval(desc)
+        nf_id = desc_dict['network_function_id']
+        return nf_id
+
+    #TODO(jiahao): Argument allocate_vip and
+    # delete_vip_port are not implememnted.
```

Comment:
Can we specify what will be missed if we don't implement the allocate_vip and delete_vip_port.